Test/improve test covrage in repository

itsahmedsalah opened the pull request.

itsahmedsalah committed:
Refactor: Improve test coverage in MediaRepositoryImplTest by adding new test cases and enhancing existing ones

itsahmedsalah committed:
Refactor: Rename AuthenticationRepositoryImplTest to UserRepositoryImplTest for clarity

itsahmedsalah committed:
Refactor: Simplify UserRepositoryImpl methods and enhance test coverage in UserRepositoryImplTest

itsahmedsalah committed:
Refactor: Enhance test coverage in SettingRepositoryImplTest by adding new test cases

itsahmedsalah committed:
Merge remote-tracking branch 'origin/develop' into test/improve-test-covrage-in-repository

Joseph-Sameh-0 reviewed (approved):
update the release info

Joseph-Sameh-0 committed:
Merge branch 'develop' into test/improve-test-covrage-in-repository

M-Abdelnabi committed:
Update release-info.txt for version 1.0.7

Increment versionName from 1.0.6 to 1.0.7.
Update release notes to reflect improved test coverage in repositories.
